scripts/client/lint-rust.mjs

```diff
@@ -1,7 +1,13 @@
 #!/usr/bin/env zx
-import 'zx/globals';
-import { getClippyToolchain, workingDirectory } from '../utils.mjs';
+import "zx/globals";
+import {
+  getClippyToolchain,
+  getToolchainArg,
+  workingDirectory,
+} from "../utils.mjs";
 
-// Check the client using clippy.
-cd(path.join(workingDirectory, 'clients', 'rust'));
-await $`cargo ${getClippyToolchain()} clippy ${process.argv.slice(3)}`;
+const toolchain = getToolchainArg(getClippyToolchain());
+// Check the client using rust fmt and clippy.
+cd(path.join(workingDirectory, "clients", "rust"));
+await $`cargo ${toolchain} fmt --check ${process.argv.slice(3)}`;
```

Comment:
To be honest, I prefer keeping these two steps separate. Maybe it's just me, but I run clippy all the time while developing, but not necessarily fmt. This also makes it hard to actually format your code.

With `format.mjs`, you can easily run `format.mjs` on its own to format the code, and `format.mjs --check` to check it in CI

Reply:
Good point. I was following the JS pattern of having only "lint", but I agree that it would be better to have them separate. SO now we have:
* `pnpm programs:format`
* `pnpm programs:clippy`
* `pnpm programs:lint` (which runs both)

Same variations for `clients:rust:*`.